internal/contract/types.go

```diff
@@ -25,7 +25,8 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-var errNotFound = errors.New("not found")
+// ErrNotFound is returned when a field is not found in the object.
+var ErrNotFound = errors.New("not found")
```

Comment:
```suggestion
// ErrFieldNotFound is returned when a field is not found in the object.
var ErrFieldNotFound = errors.New("field not found")
```
let's fix it up while we're already changing it